docs/hardware/devices/rak/base-boards.mdx

```diff
@@ -100,13 +104,16 @@ US Distributor - Purchase link: [Rokland](https://store.rokland.com/products/rak
     - Connector for 3.7v LiPo battery (with charge controller)
     - Connector for 5v solar panel (max 5.5v)
     - I<sup>2</sup>C, UART and BOOT headers accessible with solder contacts
-    - USB Type-C port for debugging and power
+    - USB-C port for debugging and power
   - **Screen Support**
     - OLED screen support (OLED screen sold separately)
 
 Further information on the RAK19003 can be found on the [RAK Documentation Center](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK19003/Overview/#product-description)
 
-US Distributor - Purchase link: [Rokland](https://store.rokland.com/products/rak-wireless-wisblock-mini-base-board-rak19003-ver-b-pid-306024)
+### Resources
+- Purchase links
+  - US Distribtor - [Rokland](https://store.rokland.com/products/rak-wireless-wisblock-mini-base-board-rak19003-ver-b-pid-306024)
+  - US Distribtor - [RAK Wireless](https://store.rakwireless.com/products/wisblock-base?variant=42441632252102)
```

Comment:
RAK isn't a US-based distributor, perhaps "international distributor"

Reply:
Changed to international, if we feel strongly about dist/manu I can change that too. Just because you're the manu doesn't mean you aren't a dist though.

Reply:
Could just say "US" and "international" and skip "Distributor"  Other links just say "AliExpress" however.  Coming up with a standard would be ideal.

Reply:
Thoughts on this?

```
### Resources
 - Resource 1
 - Resource n
 - Purchase links
   - US
     - Purchase link 1
     - Purchase link 2
   - International
     - Purchase link 1
     - Purchase link 2
```

Reply:
Yeah that works, I'm not picky at all about the form as long as it's consistent.  I think @rcarteraz made most or all of these pages so I'll defer to his ultimate wisdom.